sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/tests/Tests/ServerCRUDTests.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.SpringAppDiscovery.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.SpringAppDiscovery.Tests.Tests
+{
+    [TestFixture]
+    public class ServerCRUDTests : SpringAppDiscoveryManagementTestBase
+    {
+        public const string serverIp = "10.150.221.94";
+
+        public const string machineId = "test-swagger-marchine-id";
+
+        public const string machineId1 = "test-swagger-marchine-id1";
+
+        public ServerCRUDTests() : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                await CreateCommonClient();
+            }
+        }
```

Comment:
same here, we should remove this because this class inherits this method from its base